### PR TITLE
Datatables

### PR DESCRIPTION
**Pull request by eddier**
# Support for datatables

Adds support for SUI data tables with TW styling.  Be sure to gulp the semantic components to see new styles.  Full SUI Table docs available [here](http://react.semantic-ui.com/collections/table)

### CODE REVIEW

**Comment by cdaringe** [comment]
**[Storybook Hub](https://hub.getstorybook.io/)** is building storybooks for every commit in this PR.

**Latest Storybook: https://3ca38417-3ebc-4705-8e4e-d4aedb23a73c.sbook.io/**
<sub>( Visit [here](https://hub.getstorybook.io/app/github.com/Tripwire/octagon/branch/datatables/) for more info )</sub>

> _This comment is automatically added by Storybook Hub_

File: semantic/src/themes/tripwire/collections/table.variables [hunk 1]
```diff
@@ -182,8 +184,8 @@
 @sortableIconDescending: '\f0d7';
 @sortableDisabledColor: @disabledTextColor;
 
-@sortableHoverBackground: @transparentBlack;
-@sortableHoverColor: @hoveredTextColor;
+@sortableHoverBackground: @grey;
+@sortableHoverColor: #BFD5E1;
 
 @sortableActiveBackground: @transparentBlack;
 @sortableActiveColor: @selectedTextColor;
```
Comment (cdaringe) [inline semantic/src/themes/tripwire/collections/table.variables]: just to confirm.  there a couple of one-off colors defined in here.  are those good here, or do they need be defined in the palette?  not sure--just askin!

**Comment by cdaringe** [comment]
s3 is down... nooooo _!!!_  i cant upload my gif!
@eddier @antoinejuhel, the story for `Table` acts a little funky on a smaller res.  Can you advise?  Theres an extra node at the top that i'm not sure is per expectation.

**Comment by cdaringe** [comment]
otherwise, looks good! :)

**Comment by eddier** [comment]
SUI does collapse table headers as part of the standard table component - you can resize the tables to see that for display on say mobile.

File: semantic/src/themes/tripwire/collections/table.variables (hunk 1 above)
Comment (eddier) [inline semantic/src/themes/tripwire/collections/table.variables]: These are ok - the @ grey variable is the important one to standardize on here. # BFD5E1 isn't used widely in the library.

**Comment by cdaringe** [comment]
i see.  in our case, should we not also make the collapsed `thead` inverted colors?  because right now its just an empty box

File: semantic/src/themes/tripwire/collections/table.variables (hunk 1 above)
Comment (cdaringe) [inline semantic/src/themes/tripwire/collections/table.variables]: kk, cool. thx

**Comment by eddier** [comment]
I can do that, no problem.  The prototype didn't account for display on mobile so there may be other gaps.  Bring them up as you discover them and we can tackle them.

**Comment by jhegg** [comment]
Looks good to me.

**Comment by cdaringe** [comment]
2cool.

**Pull request merged**